.buildkite/scripts/integration_tests.sh

```diff
@@ -87,12 +95,11 @@ upload_safe_logs() {
         return
     fi
 
-    local gsUtilLocation=$(google_cloud_auth_safe_logs)
+    google_cloud_auth_safe_logs
```

Comment:
Calling this function as `$(google_cloud_auth_safe_logs)` make that the variable that was exported in that function (GOOGLE_APPLICATION_CREDENTIALS) was not available for the rest of the function. It seems that it creates a subshell and that variable does not exist outside of it.